### ISSUE TITLE
Auth section is gone. 

### ISSUE THREAD

**Issue by bmoe872** [darklow/django-suit#718]
django==1.11.14
django-suit==0.2.26

I'm not getting any errors with this, but after I did an upgrade from Django==1.8 to Django==1.11 the authorization section of the admin menu has seemingly disappeared.

I've verified that I am a superuser, so I'm not sure why this isn't displaying.

This is the relevant section of my django_suit_config in the settings.py:
```
SUIT_CONFIG = {
    'ADMIN_NAME': 'test',
    'SEARCH_URL': '',
    'MENU': (
        'auth',
```

Nothing is showing up for auth.

**Comment by bmoe872** [comment]
Alright - this wound up not being an issue with django-suit per-se, but I found the issue.

I'm using AdminPlus, and that was doing an autodiscover, which wasn't discovering the User.auth for whatever reason. 

After following this:
https://github.com/jsocol/django-adminplus/issues/42#issuecomment-195555272

I was able to solve my issue.

TL;DR:
adding `admin.sites.site = admin.site` after the call to `AdminSitePlus()` in my main urls.py solved the issue.
